src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        List<Ticket> tickets = new ArrayList<>(shoppingCart.getTickets());
+        Order order = new Order(tickets, shoppingCart.getUser());
+        Order add = orderDao.add(order);
+        shoppingCartService.clearShoppingCart(shoppingCart);
+        return add;
```

Comment:
`add` variable name is not good because it is a verb.
You can use already existing variable instead introducing the new one:
```suggestion
        order = orderDao.add(order);
        shoppingCartService.clearShoppingCart(shoppingCart);
        return order;
```